main_test.go

```diff
@@ -364,6 +388,28 @@ func startSentinel(port, masterName, masterPort string) (*redisProcess, error) {
 	return p, nil
 }
 
+func startSentinelWithAcl(port, masterName, masterPort string) (*redisProcess, error) {
+	process, err := startSentinel(port, masterName, masterPort)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, cmd := range []*redis.StatusCmd{
+		redis.NewStatusCmd(ctx, "ACL", "SETUSER", aclSentinelUsername, "ON", ">" + aclSentinelPassword, "-@all",
+			"+auth", "+client|getname", "+client|id", "+client|setname", "+command", "+hello", "+ping", "+role",
+			"+sentinel|get-master-addr-by-name", "+sentinel|master", "+sentinel|myid", "+sentinel|replicas",
+			"+sentinel|sentinels"),
+	} {
+		process.Client.Process(ctx, cmd)
```

Comment:
Can we instead run this command on existing sentinel servers? And then run `ACL DELUSER` to disable ACL?

Starting servers is a slow operation so we want to minimize the number of servers we need for testing.

Reply:
Yeah that seems fair, done.